zeth-contracts/contracts/BaseMixer.sol

```diff
@@ -93,35 +99,99 @@ contract BaseMixer is MerkleTreeMiMC7, ERC223ReceivingContract {
     }
 
     // ============================================================================================ //
+    // Reminder: The primary inputs are either field elements, hash digest or binary arrays.
+    // As a field element may not represent an entire hash digest, we formatted the former in two parts:
+    // a field element and aggregated the remaining bits into (a) common field element(s): the blob(s).
+    // For efficiency we also added in the blob the public values (bit arrays of 64 bits).
```

Comment:
Please clarify the comments. We need to stick to a single terminology. Here you use `the blob` to refer to the extra bits being packed together, while you refer to this as `residual_packing_bits` in the cpp code.